PWGHF/TableProducer/candidateCreatorToXiPi.cxx

```diff
@@ -256,10 +265,18 @@ struct HfCandidateCreatorToXiPi {
           auto trackParVarPi = getTrackParCov(trackPion);
 
           // reconstruct charm baryon with DCAFitter
-          int nVtxFromFitterCharmBaryon = df.process(trackCasc, trackParVarPi);
+          int nVtxFromFitterCharmBaryon = 0;
+          try {
+            nVtxFromFitterCharmBaryon = df.process(trackCasc, trackParVarPi);
+          } catch (...) {
+            LOG(error) << "Exception caught in charm DCA fitter process call!";
```

Comment:
Is this a bug in the fitter?

Reply:
Sometimes when running this task I get the error 
[1420770:hf-candidate-creator-to-xi-pi]: [19:05:52][ERROR] Exception caught: invalid track covariance 
and I saw that this was handles with try-catch in the LF framework, by adding this it never gives problems